mudlet-mapper.xml

```diff
@@ -8155,6 +8342,183 @@ end
                             <string>mmapper went outside</string>
                         </eventHandlerList>
                     </Script>
+                    <Script isActive="yes" isFolder="no">
+                        <name>Astral utilities</name>
+                        <packageName></packageName>
+                        <script>local astralExits = {
+	{5079,5175,&quot;n&quot;},
```

Comment:
To make it easier for people to maintain the script, you may want to consider using dictionaries here with well-named keys.

Future you will thank you :P